label_sync/labels.yaml

```diff
@@ -735,8 +735,8 @@ repos:
         target: both
         addedBy: label
       - color: 0052cc
-        description: Issues or PRs related to prow's artifact-uploader component
-        name: area/prow/artifact-uploader
+        description: Issues or PRs related to prow's artifactuploader component
+        name: area/prow/artifactuploader
```

Comment:
what was the reason for this change?
did some tooling complain about the label name?

Reply:
Yeah, curious about this too.

Reply:
``` prow/artifact_uploader/controller.go:17:1: don't use an underscore in package name ```
so i used the refactor button on my ide and renamed some of the folders